include/psp2/usbser.h

```diff
@@ -0,0 +1,31 @@
+
+#ifndef _PSP2_USBSER_H_
+#define _PSP2_USBSER_H_
+
+#include <psp2/types.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+
+	
+int SceUsbSerialStartForDriver(void);
+int SceUsbSerialSetupForDriver(int unk);	
+int SceUsbSerialStatysForDriver(void);
+int SceUsbSerialCloseForDriver(void);
+unsigned int SceUsbSerialget_lenForDriver(void);
+unsigned int SceUsbSerialSebdForDriver(const void *buffer, unsigned int len, int unk1, int unk2);	
+unsigned int SceUsbSerialrecvForDriver(void *buffer, unsigned int max_len, int unk1, int unk2);
```

Comment:
I realized after I made the request i forgot them. Will add them tommorow after work!